pro/src/ui-kit/form/PasswordInput/ValidationMessageList/ValidationMessageList.tsx

```diff
@@ -15,11 +15,13 @@ interface ValidationMessageListProps {
 }
 
 const ValidationMessageList = ({ name }: ValidationMessageListProps) => {
-  const [field] = useField({ name })
+  const [field, , helpers] = useField({ name })
```

Comment:
```suggestion
  const [field, helpers] = useField({ name })
```

Reply:
> 

Si on enlève la virgule, il y a une erreur lors de l'utilisation d'une méthode de helpers: `Property 'setError' does not exist on type 'FieldMetaProps<any>'.ts(2339)` (un point bizarre de formik)

Reply:
ça permet d'utiliser les helpers tout en évitant un warning ts si on ajoute `meta` sans l'utliser